packages/portal-proto/src/features/cohortBuilder/ContextBar.tsx

```diff
@@ -54,6 +54,7 @@ const ContextBar = ({
   const [summaryFields] = useState(INITIAL_SUMMARY_FIELDS);
   const [activeTab, setActiveTab] = useState<string | null>("summary");
   const [isGroupCollapsed, setIsGroupCollapsed] = useState(true);
+  const MIN_WIDTH_FOR_ICONS = 768;
```

Comment:
Would it make sense to add this to the tailwind theme as a breakpoint since its used in multiple components?

Reply:
I was thinking the same initially but since I am using this value to compare against viewport width -- I was not sure how I can do it. Do you have any suggestions?

Reply:
I just updated it. Thank you!

Reply:
Nice, looks great 👍